common/src/commonMain/kotlin/keijumt/library/api/ApiClient.kt

```diff
@@ -0,0 +1,23 @@
+package keijumt.library.api
+
+import io.ktor.client.HttpClient
+import io.ktor.client.features.json.JsonFeature
+import io.ktor.client.features.json.serializer.KotlinxSerializer
+import io.ktor.client.request.get
+import io.ktor.client.request.parameter
+import io.ktor.client.request.url
+
+internal class ApiClient(private val baseUrl: String, serializer: KotlinxSerializer) {
+    private val client = HttpClient {
+        install(JsonFeature) {
+            this.serializer = serializer
+        }
+    }
+
+    suspend inline fun <reified T> get(path: String, param: Map<String, String> = emptyMap()): T = client.get(path = path) {
```

Comment:
reified is not needed

Reply:
`reified` is needed because `HttpClient#get()` is using `reified` modifier, and access to type `T`.

Reply:
Android Studio doesn't display warning, but fail to build :confounded:

Reply:
I see. Thank you:)